test/quantization/test_quantize_fx.py

```diff
@@ -1802,6 +1802,24 @@ def forward(self, x):
 
         self.assertTrue(hasattr(m.mods2, 'qconfig'))
 
+    def test_not_used(self):
+        """ Test quantizing a not used value"""
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x):
+                x = x + x
+                x.sigmoid_()
+                return x
```

Comment:
is the returned value different from

```
x1 = x + x
x2 = torch.sigmoid(x1)
return x2
```
?

Reply:
yes, the first x1 is returned with current implementation

Reply:
btw, the example is not important here, it's for reproducing the case when observer has 0 uses. (which appears in pyper and I'm not sure why yet)